From ac33154594641a867d31c3379655e714aaadd8fc Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Thu, 31 Mar 2016 22:04:08 +0300
Subject: [PATCH 48/66] config: armv7: fix tiny build to new hard float
 toolchain 5.2.1

The new 5.2.1 toolchain needs the follwoing changes in the
armv7 tiny defconfig:
- compiler name
- GCC version
- GCC kernel headers version

Change-Id: I0fff638597bf9a8eff7a6e2a0f193caf40189108
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28721
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Lior Amsalem <alior@marvell.com>
---
 configs/mvebu_armv7_tiny_le_defconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configs/mvebu_armv7_tiny_le_defconfig b/configs/mvebu_armv7_tiny_le_defconfig
index 3c610d1..298feee 100644
--- a/configs/mvebu_armv7_tiny_le_defconfig
+++ b/configs/mvebu_armv7_tiny_le_defconfig
@@ -5,9 +5,9 @@ BR2_ARM_FPU_VFPV3=y
 BR2_TOOLCHAIN_EXTERNAL=y
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
 BR2_TOOLCHAIN_EXTERNAL_PATH=""
-BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="arm-marvell-linux-gnueabi"
-BR2_TOOLCHAIN_EXTERNAL_GCC_4_6=y
-BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_2=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="arm-marvell-linux-gnueabihf"
+BR2_TOOLCHAIN_EXTERNAL_GCC_5=y
+BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_18=y
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
 BR2_MVEBU_RELEASE_NAME="16.03.0"
 BR2_ROOTFS_OVERLAY="user_files"
-- 
1.9.1

